lib/tasks/add_default_level.rake

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+namespace :add_default_level do
+  desc 'Add a freak with a level'
+  task add_freak_in_db: :environment do
```

Comment:
The task does not create a freak with a level. It updates all existing freaks with a `level_id == nil`.

Please update the description & task name to be more clear.